lib/path_manager.c

```diff
@@ -43,6 +43,31 @@ static bool is_pm_ready(struct mptcpd_pm const *pm, char const *fname)
         return ready;
 }
 
+static int do_pm_add_addr(struct mptcpd_pm *pm,
```

Comment:
nit: because it is specific to `mptcpd_pm_add_addr*()` and not used elsewhere, should this be kept just above them?